modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -329,6 +331,43 @@ public static void exitNestedEventLoop(Object key, Object rval) {
         Toolkit.getToolkit().exitNestedEventLoop(key, rval);
     }
 
+    /**
+     * Returns a flag indicating whether the key corresponding to {@code keyCode}
+     * is in the locked or "on" state.
```

Comment:
I would write "locked, or "on", state" or "locked (or "on") state" because in the current form it can be understood as 2 different conditions.

Reply:
Agreed. This was nagging at the back of my mind when I read it, but I didn't do anything about it.